apps/site/components/Hero/index.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import Image from "next/image";
+import Typed from "react-typed";
+import discordImage from "../../public/discord.svg";
+import heroImage from "../../public/hero.png";
+import inviteImage from "../../public/invite.svg";
+import premiumImage from "../../public/logo.svg";
+import styles from "./hero.module.scss";
+import { Button } from "../Button";
+import { Divider } from "../Divider";
+import { Navbar } from "./Navbar";
+
+export const Hero = () => (
```

Comment:
Should this be renamed to somethingike `homepageHero` or `indexHero` to indicate it will only be on the index page?

Reply:
uh, there is only one page. I didn't really make the site with the idea of other pages.